contrib/ci/build_windows.sh

```diff
@@ -100,7 +97,6 @@ find $MINGW32BINDIR \
 	-o -name libgmp-10.dll \
 	-o -name libgnutls-30.dll \
 	-o -name libgobject-2.0-0.dll \
-	-o -name libgusb-2.dll \
```

Comment:
probably don't want to drop this do you?

Reply:
I think we can!

Reply:
Oh I misread, yeah!